swagger.yaml

```diff
@@ -1,42 +1,233 @@
 swagger: "2.0"
+
 info:
   title: dp-search-reindex-api
   description: "Provides details about search reindex jobs and enables creation and running of them"
   version: 1.0.0
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "search-reindex"
```

Comment:
usually this is either "public", private
```
tags:
- name: "Public"
- name: "Private"
```

Reply:
done